std-bits/aws/src/main/resources/META-INF/native-image/org/enso/aws/reflect-config.json

```diff
@@ -201,5 +186,16 @@
                 "parameterTypes": []
             }
         ]
+    },
+    {
+        "name": "software.amazon.awssdk.regions.Region",
+        "methods": [
```

Comment:
- Why is this needed?
- I'd expect the access to `Region` to be done via `AWSRegion` wrapper class
- then this reflection config shouldn't be needed for access from Enso to the library
- is there some left-over access left?
- or is the reflective access needed for some other reason?

Reply:
Still the case (https://github.com/enso-org/enso/pull/12743/files#diff-2c918c18cf24e0fc51f773420c73129f4d107b002ff9795692121a0bf4de81a4R40)
```
    - [FAILED] should be able to always resolve a default region to be used [6763ms]
        Reason: An unexpected panic was thrown: (No_Such_Method.Error eu-west-1 UnresolvedSymbol<id>)
        at <enso> case_branch(enso/distribution/lib/Standard/AWS/0.0.0-dev/src/AWS_Region.enso:40:13-78)
        at <enso> AWS_Region.resolve_id(enso/distribution/lib/Standard/AWS/0.0.0-dev/src/AWS_Region.enso:37-42)
```

Reply:
Rewrote the original class to avoid accessing Java method in Enso.